# Effectiveness of a New Drug for Lowering Blood Pressure

A pharmaceutical company developed a new drug intended to lower blood pressure. To determine the drug's effectiveness, a clinical trial was conducted. Researchers wanted to see if there was a statistically significant difference in blood pressure levels before and after administering the drug.

**Study Design**
- Sample Size: 100 patients
- Pre-treatment Mean Blood Pressure (µ₀): 140 mmHg
- Post-treatment Mean Blood Pressure (X̄): 135 mmHg
- Standard Deviation (σ): 15 mmHg
- Significance Level (α): 0.05

**Hypotheses**
- Null Hypothesis (H₀): The mean blood pressure after treatment is equal to or greater than the mean blood pressure before treatment (µ₁ ≥ µ₀).
- Alternative Hypothesis (H₁): The mean blood pressure after treatment is less than the mean blood pressure before treatment (µ₁ < µ₀).



### Program to solve this problem 

In [3]:
import scipy.stats as stats
import numpy as np

# Simulated data for pre-treatment and post-treatment blood pressure
np.random.seed(0)  # For reproducibility

# Sample size
n = 100

# Generate pre-treatment blood pressure data (assuming normal distribution)
pre_treatment_data = np.random.normal(loc=140, scale=15, size=n)

# Generate post-treatment blood pressure data (assuming normal distribution)
post_treatment_data = np.random.normal(loc=135, scale=15, size=n)

# Optional: Print some of the data
print("\nSample Data (First 10 values):")
print("Pre-treatment Data:", pre_treatment_data[:10])
print("Post-treatment Data:", post_treatment_data[:10])
print()

# Calculate sample means
mu_0 = np.mean(pre_treatment_data)
x_bar = np.mean(post_treatment_data)

# Known population standard deviation (σ)
sigma = 15  # As per the given case study

# Significance level
alpha = 0.05

# Calculate the standard error
SE = sigma / np.sqrt(n)

# Calculate the z-value
z_value = (x_bar - mu_0) / SE

# Calculate the p-value
p_value = stats.norm.cdf(z_value)

# Print the results
print(f"Sample Size (n): {n}")
print(f"Pre-treatment Mean Blood Pressure (µ₀): {mu_0:.2f} mmHg")
print(f"Post-treatment Mean Blood Pressure (X̄): {x_bar:.2f} mmHg")
print(f"Standard Deviation (σ): {sigma} mmHg")
print(f"Significance Level (α): {alpha}")
print(f"Standard Error (SE): {SE:.2f}")
print(f"Calculated z-value: {z_value:.2f}")
print(f"Calculated p-value: {p_value:.5f}")

# Decision
if p_value < alpha:
    print("Reject the null hypothesis (H₀).")
    print("Conclusion: The new drug effectively lowers blood pressure.")
else:
    print("Fail to reject the null hypothesis (H₀).")
    print("Conclusion: There is no significant evidence to suggest the new drug lowers blood pressure.")





Sample Data (First 10 values):
Pre-treatment Data: [166.46078519 146.00235813 154.68106976 173.61339799 168.01336985
 125.3408318  154.25132626 137.72964188 138.45171722 146.15897753]
Post-treatment Data: [163.24726046 114.78361408 115.94272502 149.54095062 117.40314892
 164.15431778 128.79571529 123.78817783 163.8441304  157.20772187]

Sample Size (n): 100
Pre-treatment Mean Blood Pressure (µ₀): 140.90 mmHg
Post-treatment Mean Blood Pressure (X̄): 136.23 mmHg
Standard Deviation (σ): 15 mmHg
Significance Level (α): 0.05
Standard Error (SE): 1.50
Calculated z-value: -3.11
Calculated p-value: 0.00093
Reject the null hypothesis (H₀).
Conclusion: The new drug effectively lowers blood pressure.


**Interpretation**
- The z-value of -3.11 indicates how many standard deviations the observed mean (136.23 mmHg) is below the hypothesized mean (140.90 mmHg).
- Using the standard normal distribution table, a z-value of -3.11 corresponds to a very low p-value 0.00093 (much less than 0.05).

Since the p-value is lower than the significance level (α = 0.05), the researchers reject the null hypothesis.